pkg/streamingpromql/functions/common_test.go

```diff
@@ -43,6 +44,8 @@ func TestFloatTransformationFunc(t *testing.T) {
 			{H: &histogram.FloatHistogram{Count: 1, Sum: 2}},
 		},
 	}
+	// Increase the memory tracker for 2 FPoints, and 1 HPoint
+	memoryConsumptionTracker.IncreaseMemoryConsumption(types.FPointSize*2 + types.HPointSize*1)
```

Comment:
Why is this necessary? Can't we just assert that `CurrentEstimatedMemoryConsumptionBytes` is 0 below?

(same comment applies to the other similar changes in this file)

Reply:
Because the functions we're testing actually call `DecreaseMemoryConsumption` at some point, which requires the current estimated memory to be larger than what is returned. I figured may as well put in real values if that path is being executed.

Reply:
Sorry, I'm missing something: where is `DecreaseMemoryConsumption` called in this test? `floatTransformationFunc` reuses the slice it's passed.

(I can see this is necessary for `TestFloatTransformationDropHistogramsFunc` though)

Reply:
oh, sorry, I see your point. I'm doing this for consistency with `TestFloatTransformationDropHistogramsFunc`